djangocms_text_ckeditor/static/djangocms_text_ckeditor/js/cms.ckeditor.js

```diff
@@ -214,19 +400,78 @@
                     // in case there are multiple different inlines we need to check
                     // newly added one against all of them
                     dynamics.forEach(function (config) {
-                        var selector = config[0];
+                        var selector = config[0].id;
                         var regex = new RegExp(selector.replace('__prefix__', '\\d+'));
 
                         if (containerId.match(regex)) {
-                            CMS.CKEditor.init(containerId, config[1], config[2]);
+                            CMS.CKEditor.init(
+                                document.getElementById(containerId),
+                                options,
+                                settings
+                            );
                         }
                     });
                 });
             });
+        },
+
+        _highlight_Textplugin: function (pluginId) {
```

Comment:
I intercept the pointerover event to avoid the tooltip to appear (which says "Double-click to edit"). Firstly, this is incorrect when editing inline, secondly the tooltip covers the text to be edited which is irritating. I would not have to rebuild "highlight on click" if I could tell the cms js-frontend just not to show the tooltip for text plugins. Would you know if there is a way to achieve this? Looking at the tooltip source it seems this cannot be steered on a plugin level.